arbitrary-users-patch/base_images

```diff
@@ -1 +1,10 @@
-java11-maven maven:3.6.0-jdk-11
+che7-python-3.6          centos/python-36-centos7:1
```

Comment:
why `che7-*` and not just `che-*`?

Reply:
It's mostly a holdover from when I intended these changes to go in the `che-dockerfiles` repo; I'll switch to plain `che-`.